### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by skofman1**

### CODE REVIEW

File: tests/Monitoring.RebootSearchInstance.Tests/Monitoring.RebootSearchInstance.Tests.csproj [hunk 1]
```diff
@@ -60,6 +60,9 @@
     <PackageReference Include="Moq">
       <Version>4.7.145</Version>
     </PackageReference>
+    <PackageReference Include="System.Text.RegularExpressions">
```
Comment (scottbommarito) [inline tests/Monitoring.RebootSearchInstance.Tests/Monitoring.RebootSearchInstance.Tests.csproj]: Why is this required?

Comment (skofman1) [reply tests/Monitoring.RebootSearchInstance.Tests/Monitoring.RebootSearchInstance.Tests.csproj]: we get a build warning about trying to find version 4.0.12 and choosing 4.1.0

Comment (skofman1) [reply tests/Monitoring.RebootSearchInstance.Tests/Monitoring.RebootSearchInstance.Tests.csproj]: Warning	NU1603	Microsoft.IdentityModel.Clients.ActiveDirectory 3.13.5 depends on System.Text.RegularExpressions (>= 4.0.12) but System.Text.RegularExpressions 4.0.12 was not found. An approximate best match of System.Text.RegularExpressions 4.1.0 was resolved.	Monitoring.RebootSearchInstance.Tests	C:\Nuget\NuGet.Jobs\tests\Monitoring.RebootSearchInstance.Tests\Monitoring.RebootSearchInstance.Tests.csproj	1
